src/Uno.UWP/UI/Composition/Compositor.Skia.cs

```diff
@@ -44,7 +45,8 @@ public void Dispose()
 
 		internal void Render(SKSurface surface, SKImageInfo info)
 		{
-			var sw = Stopwatch.StartNew();
+			_isDirty = false;
+			// var sw = Stopwatch.StartNew();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6287442)